.travis.yml

```diff
@@ -17,6 +17,7 @@ script:
     -e TEST_DB_PORT=3306
     -e BRANCH=$TRAVIS_BRANCH
     -e COMMIT_ID=$TRAVIS_COMMIT
+    -e STAGE=production
```

Comment:
STAGE could be misinterpreted as a "staging" environment (which is what just happened with me). A better name of this variable could be "ENVIRONMENT" or "ENV"